engine/src/witness/btc.rs

```diff
@@ -56,7 +59,9 @@ pub async fn process_egress<ProcessCall, ProcessingFut, ExtraInfo, ExtraHistoric
 					signer_id: DepositAddress::new(epoch.info.0.current, CHANGE_ADDRESS_SALT)
 						.script_pubkey(),
 					tx_fee: tx.fee.unwrap_or_default().to_sat(),
-					tx_metadata: (),
+					tx_metadata: BitcoinTransactionMetadata {
+						tx_hash: tx_hash.iter().rev().map(|elem| elem.to_owned()).collect_array(),
```

Comment:
Probably better to have a type to do this - then we can write tests too, to see that the bytes display the correct way